templates/addCourse.phtml

```diff
@@ -26,6 +26,12 @@ $trainersCheckbox = \Portal\ViewHelpers\TrainerCheckboxViewHelper::displayTraine
         <label for="course-name">Course Name</label>
         <input name="courseName" class="create-courses form-control" type="text" placeholder="Course name" data-required="true" data-max='255' maxlength="255" />
         <div id="courseNameError" class="alert hidden formItem_alert"></div>
+        <label for="course-category">Category</label>
```

Comment:
for attribute has to match the ID of the select, remove the `-` and capitalise `Category`